src/pudl/glue/eia_epacems.py

```diff
@@ -0,0 +1,125 @@
+"""
+Define and fill in gaps of the EPA-EIA crosswalk.
+
+This module defines functions that read in the EPA-EIA crosswalk
+and fill in as many of the eia id gaps as possible. Eventually, EPA
+is going to come out with a crosswalk file containing fewer gaps.
+Until then, this module reads and cleans the current crosswalk.
+"""
+import importlib
+import logging
+
+import pandas as pd
+
+import pudl
+
+logger = logging.getLogger(__name__)
+
+
+def grab_n_clean_epa_orignal():
+    """Retrieve the original EPA-EIA crosswalk file."""
+    logger.info("grabbing original crosswalk")
+    eia_epacems_crosswalk_csv = (
+        importlib.resources.open_text(
+            'pudl.package_data.glue',
+            'epa_eia_crosswalk_from_epa.csv')
+    )
+    eia_epacems_crosswalk = (
+        pd.read_csv(eia_epacems_crosswalk_csv)
+        .pipe(pudl.helpers.simplify_columns)
+        .rename(columns={
+            'oris_code': 'plant_id_epa',
+            'eia_oris': 'plant_id_eia',
+            'unit_id': 'epa_point_source_unit',
+            'facility_name': 'plant_name_eia',
+            'unit_type': 'prime_mover_code'})
+        .drop([
+            'fuel_type_primary',
+            'edat_mw_cap',
+            'way_gen_id_matched',
+            'unit_op_status_date',
+            'notes',
+            'annual_heat_input',
+            'op_status'], axis=1)
```

Comment:
The pattern of dropping the columns you don't want, rather than explicitly keeping only the columns you do want, is vulnerable to changes in the column names or the inclusion of new columns, which would mean the dataframe has unexpected columns in it at the end of the function. We definitely did this a lot early on in our transform function writing, but have tried to move toward having the dataframes be more reliably of a particular form at the end.

Reply:
Good bit of advice - I switched it to `filter` vs `drop`